app/navigation/InformationNavigator.scala

```diff
@@ -47,18 +48,18 @@ class InformationNavigator @Inject()() extends BaseNavigator {
       (userAnswers: UserAnswers) => userAnswers.get(ChangeDestinationTypePage) match {
         case Some(wantsToChangeDestinationType) =>
           if(wantsToChangeDestinationType) {
-            controllers.sections.info.routes.DestinationTypeController.onPreDraftPageLoad(userAnswers.ern, userAnswers.arc, NormalMode)
+            controllers.sections.info.routes.NewDestinationTypeController.onPreDraftPageLoad(userAnswers.ern, userAnswers.arc)
           } else {
             controllers.routes.DraftMovementController.onPageLoad(userAnswers.ern, userAnswers.arc)
           }
         case _ => controllers.sections.info.routes.ChangeDestinationTypeController.onPageLoad(userAnswers.ern, userAnswers.arc)
       }
 
     case DispatchPlacePage =>
-      (userAnswers: UserAnswers) => controllers.sections.info.routes.DestinationTypeController.onPreDraftPageLoad(userAnswers.ern, userAnswers.arc, NormalMode)
+      (userAnswers: UserAnswers) => controllers.sections.info.routes.NewDestinationTypeController.onPreDraftPageLoad(userAnswers.ern, userAnswers.arc)
```

Comment:
DispatchPlacePage shouldn't be used for routing now the controller has been deleted